atlp-rwanda/Team-furebo-E-commerce-fn
# Ft user registration #185172178

What does this PR do?

- This PR is for registering a user and verify email after registering

How should this be manually tested?

- clone the repostory
- cd into the name of the repostory
- npm install (to install the dependencies)
- npm start (to start the app)
- navigate to signup
- enter your creadentials
- then click signup
- go to your email and verify your email

What are the relevant pivotal tracker/Trello stories?

- #185172178
